modules/klump/klump.php

```diff
@@ -23,7 +23,7 @@ public function __construct()
     {
         $this->name = 'klump';
         $this->tab = 'payments_gateways';
-        $this->version = '1.0.0';
+        $this->version = '0.1.0';
```

Comment:
Why not 1.0.0?
@cyberomin

Reply:
Yeah but it should be 1.1.0 not 0.1.0.
@cyberomin

Reply:
@richienabuk ```
Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.``` Also, there is a bug in this code that isn't fixed yet.